openforcefield/topology/molecule.py

```diff
@@ -1890,55 +1894,119 @@ def from_smiles(smiles,
                 'Invalid toolkit_registry passed to from_smiles. Expected ToolkitRegistry or ToolkitWrapper. Got  {}'
                 .format(type(toolkit_registry)))
 
-    def is_isomorphic(
-            self, other,
-            compare_atom_stereochemistry=True,
-            compare_bond_stereochemistry=True,
+    @staticmethod
+    def are_isomorphic(
```

Comment:
Why go with a static method instead of `.is_isomorphic_with(other)`?

Reply:
I think the staticmethod is best here, since it will give us a single function to handle `FrozenMolecule`s, `TopologyMolecule`s, and `nx.Graph`s. So, we'll still have `FrozenMolecule.is_isomorphic_with(other)`, but it can just immediately call this single, flexible method, which will sort out what `other` is and how to efficiently check it.

Reply:
Added the `.is_isomorphic_with(other)` function which now calls the static function, not sure what level of checking we should be doing here though

Reply:
> not sure what level of checking we should be doing here though

We can add `**kwargs` to the `is_isomorphic_with` function signature, and just pass them all on to `are_isomorphic`, though we will want to update the docstring for that funciton to reflect the change. I've added another comment about this at the appropriate line.